activation/activationdb_test.go

```diff
@@ -331,6 +331,56 @@ func Test_DBSanity(t *testing.T) {
 	assert.Equal(t, *types.EmptyATXID, id)
 }
 
+func Test_DBGetByCoinbase(t *testing.T) {
+	atxdb, _, _ := getAtxDb("t6")
+
+	id1 := types.NodeID{Key: uuid.New().String()}
+	coinbase1 := types.HexToAddress("aaaa")
+	coinbase2 := types.HexToAddress("bbbb")
+	coinbase3 := types.HexToAddress("0")
+
+	atx1 := newActivationTx(id1, 0, *types.EmptyATXID, 3, 0, *types.EmptyATXID, coinbase1, 3, []types.BlockID{}, &types.NIPST{})
+	atx2 := newActivationTx(id1, 0, *types.EmptyATXID, 1001, 0, *types.EmptyATXID, coinbase2, 3, []types.BlockID{}, &types.NIPST{})
+	atx3 := newActivationTx(id1, 0, *types.EmptyATXID, 2001, 0, *types.EmptyATXID, coinbase1, 3, []types.BlockID{}, &types.NIPST{})
+
+	atxid1 := atx1.ID()
+	atxid2 := atx2.ID()
+	atxid3 := atx3.ID()
+
+	err := atxdb.storeAtxUnlocked(atx1)
+	assert.NoError(t, err)
+	err = atxdb.storeAtxUnlocked(atx2)
+	assert.NoError(t, err)
+	err = atxdb.storeAtxUnlocked(atx3)
+	assert.NoError(t, err)
+
+	// DB stores atxs sorted using byte by byte comparison of atxids. Thus 3 > 2001 since 3 > 2
```

Comment:
This does not appear to be the case right now (this test is failing). Why would the database sort things, i.e., not continue to store them in the order they were originally stored?

Reply:
LevelDB stores values sorted by key: https://github.com/google/leveldb#features (see 2nd bullet).